tests/expected/issue-3631/issue-3631.expected

```diff
@@ -0,0 +1 @@
+VERIFICATION:- SUCCESS
```

Comment:
Can you please move this test to the `kani` suite? We use expected when we care about validating the output.